fmralign/tests/test_alignment_methods.py

```diff
@@ -151,7 +159,7 @@ def test_all_classes_R_and_pred_shape_and_better_than_identity():
                      ScaledOrthogonalAlignment(scaling=False),
                      OptimalTransportAlignment(),
                      Hungarian(), DiagonalAlignment()]:
-            print(algo)
+            # print(algo)
```

Comment:
Should we set up a logger with a `DEBUG` level for these sorts of statement ? Not for this PR, just thinking about whether it's worth opening an issue.

Reply:
maybe not yet, for now the biggest and almost only sources of bugs is clustering labels for various reasons (which is accessible through verbose)